golem-cli/src/component.rs

```diff
@@ -112,6 +123,18 @@ impl<C: ComponentClient + Send + Sync> ComponentHandler for ComponentHandlerLive
 
                 Ok(GolemResult::Ok(Box::new(views)))
             }
+            ComponentSubCommand::Get {
+                component_id_or_name,
+                version,
+            } => {
+                let component_id = self.resolve_id(component_id_or_name).await?;
+                let component = match version {
+                    Some(v) => self.get_metadata(&component_id, v).await?,
+                    None => self.get_latest_metadata(&component_id).await?,
+                };
+                let view: ComponentView = component.into();
+                Ok(GolemResult::Ok(Box::new(vec![view])))
```

Comment:
Why `vec!`? Please create `ComponentGetView` similar to `ComponentAddView`. Also please add tests both in `component.rs` and `text.rs`

Reply:
You can just copy `update` tests in both with `get` instead of `update`